src/components/NavFooter.vue

```diff
@@ -1,7 +1,90 @@
 <template>
-    <div>
-        ugracing@uoguelph.ca<br />
-        50 Stone Road East, Guelph, N1G2W1<br />
-        (519) 824-4120 ext 52698
+    <div class="footerWrapper">
+        <div class="mainFooter">
+            <!-- insert social media images here with links: -->
+            <div class="socialMediaLogos">
+                <h2>Follow us</h2>
+                <br />
+                <ul>
+                    <li>
+                        <img />
+                    </li>
+                    <li>
+                        <img />
+                    </li>
+                    <li>
+                        <img />
+                    </li>
+                    <li>
+                        <img />
+                    </li>
+                </ul>
+            </div>
+
+            <div class="teamLogo">
+                <NuxtLink to="/"><img src="../../public/media/GryphonBanner.svg" /></NuxtLink>
```

Comment:
```suggestion
                <RouterLink to="/"><img src="/media/GryphonBanner.svg" /></RouterLink>
```
not using nuxt anymore